src/containers/Dashboard/Navbar.js

```diff
@@ -1,13 +1,17 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
-import { HOME } from "../../constants";
+import { DASHBOARD } from "../../constants";
 import "./Navbar.scss";
 class Navbar extends Component {
+  logout = () => {
+    localStorage.clear();
```

Comment:
@Humnaa 
Create separate service for Session and move login/logout implementation there.

Reply:
Create separate folder `service` and move session related implementation there